neos/redirecthandler-ui
# BUGFIX: Prevent empty list with preselected code/type

When a status code or type is configured as preselected
but it has no results, also no other redirects would be shown initially.
This change resets the preset when there is no match.

How to test:

Set `initialStatusCodeFilter` in the settings to 999 and reload the redirect list.
Without the patch the list would be empty. With the patch it should show all results.

Set `initialTypeFilter` in the settings to "foo" and reload the redirect list.
Without the patch the list would be empty. With the patch it should show all results.
